examples/locality-aware-routing/README.md

```diff
@@ -0,0 +1,143 @@
+  for all resources created by the example. The examples below use the name `ecs`.
+- You need to pass in the IP address of your workstation via the `lb_ingress_ip`
+  variable. This is used for the security groups on the elastic load balancers to ensure
+  only you have access to them.
+- Consul Enterprise license that needs to be passed to the `consul_license` variable. This license will be used to run the enterprise version of Consul in this example. One way to pass these licenses would be add them to a `.tfvars` file and pass it as an argument to the `terraform apply` command.
+
+Determine your public IP. You can use a site like https://ifconfig.me/:
+
+```console
+$ curl ifconfig.me
+123.456.789.1%
+```
+
+Example `input.tfvars`
+
+```
+lb_ingress_ip  = "123.456.789.1"
+consul_license = "<license>"
+```
+
+Initialize Terraform:
+
+```console
+$ terraform init
+```
+
+### Terraform Apply
+
+Then apply the Terraform passing in a name and your IP:
+
+```console
+$ terraform apply \
+    -var name=ecs \
+    -var-file=input.tfvars
+```
+
+The plan should look similar to:
+
+```shell
+Plan: 81 to add, 0 to change, 0 to destroy.
+
+Changes to Outputs:
+  + client_lb_address             = (known after apply)
+  + consul_server_bootstrap_token = (sensitive value)
+  + consul_server_url             = (known after apply)
+```
+
+Type `yes` to apply the changes.
+
+~> **Warning:** These resources will cost money. Be sure to run `terraform destroy`
+   when you've finished testing.
+
+The apply should take 7-10 minutes. When complete, the URLs of the two load
+balancers should be in the output, along with the bootstrap token for the Consul server:
+
+```shell
+Apply complete! Resources: 81 added, 0 changed, 0 destroyed.
+
+Outputs:
+
+client_lb_address = "http://example-client-app-1959503271.us-west-2.elb.amazonaws.com:9090/ui"
+consul_server_bootstrap_token = <sensitive>
+consul_server_url = "http://ecs-dc1-consul-server-713584774.us-west-2.elb.amazonaws.com:8500"
+```
+
+### Explore
+
+Get the bootstrap token for the Consul server from the Terraform output:
+
+```console
+$ terraform output -json | jq -r .consul_server_bootstrap_token.value
+e2cb39e2-b9fd-18af-025f-86f6da6889a7
+```
+
+If you click on the URL of the `consul_server_url`, you should be able
+to view the Consul UI and log in using the `consul_server_bootstrap_token` above:
+
+![Consul dc1 UI](https://github.com/hashicorp/terraform-aws-consul-ecs/blob/main/_docs/locality-aware-dc1-ui.png?raw=true)
+
+If you browse to the URL of the `client_lb_address`, the example application UI should be displayed:
+
+![Example App UI](https://github.com/hashicorp/terraform-aws-consul-ecs/blob/main/_docs/locality-aware-dc1-ui.png?raw=true)
+
+Notice the IP of the upstream server application's task. Because of the locality parameters added during the service registration, Consul takes care of routing traffic from the client application to the server application task within the same availability zone.
```

Comment:
WDYT about adding:
- a code snippet with an example of the applicable LAR configuration; and/or,
- a link to the Consul docs?